pyam/iiasa.py

```diff
@@ -558,7 +559,7 @@ def read_iiasa(name, default=True, meta=True, creds=None, base_url=_AUTH_URL, **
     ----------
     name : str
         A valid name of an IIASA scenario explorer instance,
-        see :attr:`pyam.iiasa.Connection.valid_connections`
+        see :attr:`pyam.iiasa.Connection().valid_connections`
```

Comment:
Did you test whether the automated API documentation correctly links with the added brackets?